styles/utility.css

```diff
@@ -329,14 +331,17 @@ div.contact-me {
 .contact-form {
   margin-top: 40px;
   width: 100%;
+  display: flex;
+  flex-flow: column;
+  align-items: center;
 }
 
 .form-group {
   width: 100%;
   height: 48px;
-  border-radius: 6px;
+  border-radius: 4px;
   padding: 6px;
-  margin: 24px 0;
+  margin: 15px 0;
 }
 
 .form-control {
```

Comment:
- [ ] You still have to fix styling for inputs. I also suggest you have a look at the Figma template requirement a little bit more carefully. Here I will write down from the template the code you should to add in your `.form-control` class: 
```
border: 1px solid #D0D9D4;
box-sizing: border-box;
border-radius: 4px;
```
- I did it and it looks very pretty

<img width="278" alt="Screen Shot 2022-01-20 at 19 20 27" src="https://user-images.githubusercontent.com/49064106/150367805-9e9d429e-5772-4977-9756-ab2edf92d3bd.png">

Reply:
Hi @elmar8287 
I have adjusted the contact suggestion just as suggested.
Thanks for your review 👍